--- Makefile_old	2015-08-21 15:43:53.769717804 -0700
+++ Makefile_change	2015-08-21 15:43:05.130325915 -0700
@@ -1,13 +1,13 @@
 all:
 	make os=linux project=minizip -C dom
 	make os=linux project=dom -C dom
-	make os=linux project=fx -C fx
-	make os=linux project=rt -C rt
-	make os=linux project=viewer -C viewer
+#	make os=linux project=fx -C fx
+#	make os=linux project=rt -C rt
+#	make os=linux project=viewer -C viewer
 	
 clean:
 	make os=linux project=minizip -C dom clean
 	make os=linux project=dom -C dom clean
-	make os=linux project=fx -C fx clean
-	make os=linux project=rt -C rt clean
-	make os=linux project=viewer -C viewer clean
+#	make os=linux project=fx -C fx clean
+#	make os=linux project=rt -C rt clean
+#	make os=linux project=viewer -C viewer clean
